drivers/sensors/cirque_pinnacle.c

```diff
@@ -204,40 +201,89 @@ void cirque_pinnacle_init(void) {
 #endif
 
     touchpad_init = true;
+
     // Host clears SW_CC flag
     cirque_pinnacle_clear_flags();
 
-    // Host configures bits of registers 0x03 and 0x05
-    RAP_Write(SYSCONFIG_1, SYSCONFIG_1_VALUE);
-    RAP_Write(FEEDCONFIG_2, FEEDCONFIG_2_VALUE);
-
-    // Host enables preferred output mode (absolute)
-    RAP_Write(FEEDCONFIG_1, FEEDCONFIG_1_VALUE);
+    // SysConfig1 (Low Power Mode)
+    // Bit 0: Reset, 1=Reset
+    // Bit 1: Shutdown, 1=Shutdown, 0=Active
+    // Bit 2: Sleep Enable, 1=low power mode, 0=normal mode
+    // send a RESET command now, in case QMK had a soft-reset without a power cycle
+#if CIRQUE_PINNACLE_RESET_ON_INIT_WAIT > 0
+    RAP_Write(SYSCONFIG_1, 0x01);
+    wait_ms(CIRQUE_PINNACLE_RESET_ON_INIT_WAIT);
+#endif
```

Comment:
> Pinnacle needs 10-15ms to boot, so wait long enough before configuring

This was the comment in the first the commit. Is there any reason not to wait? If not I would drop this option entirely as 30ms are hardly noticeable.

Reply:
I only had two data points: mine was happy with only 5ms, while another user reported issues with <15ms. I therefore chose a reasonably higher safe value. I kept it as an option for advanced users to really tweak and speed-boot-tune their system if needed.

Reply:
The chosen timeout value sounds reasonable to me :+1:

I'm in favor in keeping this as a fixed value and not expose it to be configurable. Reason is that configuration value increases the complexity of the user facing API for little in return, as the gain-able 25ms difference will hardly be noticeable at all.

Reply:
fixed value as in not even a define, or just a define at the top of the .c file?

Reply:
The first commit of this PR with the comment explaining the need for the chosen value would suffice from my POV.

Reply:
Done!